src/AgentModel.cpp

```diff
@@ -410,8 +410,9 @@ void AgentModel::decisionProcessStop() {
         // add stop point because of target
         if (stop)
         {
+            // try to stop 10m before intersection
             double ds_stop = std::max(0.0, _input.vehicle.dsIntersection - 10);
-            _state.decisions.stopping[1].id = agent_model::NOS + 1; //convention
+            _state.decisions.stopping[1].id = -1;
```

Comment:
id is an unsinged int

Reply:
https://github.com/ika-rwth-aachen/SimDriver/blob/d57fc09c81abe4f34aa52f372a97d1ffd1fe1067/src/Interface.h#L182